### ISSUE TITLE
🐛 After updating to 0.8.1, the command `delta` no longer work when any file contains spaces

### ISSUE THREAD

**Issue by NariyasuHeseri** [dandavison/delta#647]
Suppose there are two files `file1` and `file 1` in the current directory.
Either `delta file file\ 1` or `delta file "file 1"` will produce the following output:
```
error: Found argument '1' which wasn't expected, or isn't valid in this context

USAGE:
    delta [FLAGS] [OPTIONS] [ARGS]

For more information try --help
```
It is likely that the bug is caused by commit `ac2873dabe53916354f264c7a9b26a18a23c082b`.

**Comment by dandavison** [comment]
Hi @NariyasuHeseri, yes, sorry about that. This is a dup of https://github.com/dandavison/delta/issues/644 and it's fixed in master. Perhaps I should do another bug fix release.

**Comment by dandavison** [comment]
That's released in 0.8.2

**Comment by NariyasuHeseri** [comment]
> That's released in 0.8.2

Thanks!
